src/core/ext/filters/client_channel/lb_policy/pick_first/pick_first.cc

```diff
@@ -94,11 +83,7 @@ class PickFirstConfig : public LoadBalancingPolicy::Config {
   }
 
   void JsonPostLoad(const Json& /* json */, const JsonArgs& /* args */,
```

Comment:
This method can be removed.  It doesn't need to be provided if it's a no-op.

Reply:
Done. Please take another look. Thanks!